tests/MonoGame.Extended.Tests/MockGameWindow.cs

```diff
@@ -10,15 +10,19 @@ public class MockGameWindow : GameWindow
     {
         public override bool AllowUserResizing { get; set; }
         public override Rectangle ClientBounds { get; }
-        public override Point Position { get; set; }
         public override DisplayOrientation CurrentOrientation { get; }
         public override IntPtr Handle { get; }
         public override string ScreenDeviceName { get; }
 
+#if !KNI
+        public override Point Position { get; set; }
+#endif
+
         public MockGameWindow()
         {
         }
 
+#if !KNI
         public override void BeginScreenDeviceChange(bool willBeFullScreen)
```

Comment:
BeginScreenDeviceChange(), EndScreenDeviceChange(), SetSupportedOrientations() are implementation methods in XNA, not to be used by end users.  They were removed in KNI.

Reply:
I don't see where we're using these in the MGE tests, if they can be removed just remove them.

Reply:
There is a reference in BoxingViewportAdapterTests.cs
which is commented out since 3.7
https://github.com/craftworkgames/MonoGame.Extended/commit/bcd91713177fed8a089d14167f058a286f39ed27#diff-b89e85a38a6542fae70bdaaf960e62b3e398d0de24d3e1d9dec5e70071083be0

I'll remove both. 
BoxingViewportAdapter is a graphics class. It doesn't need a Window. It can do the same from graphicsDevice.PresentationParameters and GraphicsReset event.

Reply:
Done.